test/factories.ts

```diff
@@ -206,15 +175,20 @@ export const TYPE_GENERATORS: ITypeGenerators = {
   date: { valueFunction: fakeDateRecent, fieldConfigFactory: dateFactory },
```

Comment:
The next step for this testing improvement would be to have a fill in function for `TYPE_GENERATORS` that would automatically set the `fieldConfigFactory` to `fieldFactoryForType(FIELD_NAME)` so factory declarations wouldn't be needed except in special cases

Reply:
Worth considering.